### ISSUE TITLE
Stop at validation sanity check

### ISSUE THREAD

**Issue by xuanxu92** [IceClear/StableSR#80]
Hi, I changed to my dataset to train SFT and it freezes at validation sanity check

Could you please tell me how to solve this issue?

thanks

**Comment by IceClear** [comment]
Hi. Please refer to #44 .

**Comment by xuanxu92** [comment]
thanks!

**Comment by Sunlight-qwq** [comment]
Hello. I met the same problem. The program has been stuck at validation sanity check for over 12 hours, and no new output is printed. Is it a normal situation?

**Comment by IceClear** [comment]
there should be something wrong. most likely to be the same issue as above.

**Comment by Sunlight-qwq** [comment]
> there should be something wrong. most likely to be the same issue as above.

Could you please tell us the dataset and file structure that you used to train this model? I wanna test on the origin dataset to find out where something wrong occurred.

**Comment by IceClear** [comment]
> Hi. Please refer to #44 .

here

**Comment by Sunlight-qwq** [comment]
> > Hi. Please refer to #44 .
> 
> here

Thanks. After annotating this while block, the training code runs normally.
Would this code be modified in the future? I do not think judging an image according to its size is a good strategy. Maybe it can be an optional function?

**Comment by IceClear** [comment]
Thanks for the suggestion. I may improve this when I am avaliable.

**Comment by H0128Z** [comment]
> > > Hi. Please refer to #44 .
> > 
> > 
> > here
> 
> Thanks. After annotating this while block, the training code runs normally. Would this code be modified in the future? I do not think judging an image according to its size is a good strategy. Maybe it can be an optional function?

I agree with @Sunlight-zero 's suggestions. I'd had a same problem. 
@IceClear  May I ask for what purpose this block(https://github.com/IceClear/StableSR/issues/44) was created?
